azure/terraform/README.md

```diff
@@ -0,0 +1,94 @@
+# A minimal viable Metaflow-on-Azure stack
+
+## What does it do?
+It provisions all necessary Azure resources. Main resources are:
+* Azure Blob Storage account + container
+* AKS cluster
+It will also deploy Metaflow services onto the AKS cluster above.
+
+## Prerequisites
+
+* Install [terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli).
+* Install [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl).
+* Install [az](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) CLI.
+* `az login` as an Azure account with sufficient privileges to administer necessary resources:
+  * AKS
+  * Azure storage accounts and containers
+  * Virtual networks
+  * AAD applications
+  * ...
+  * Note-1: If you are an admin of your Azure subscription already you should be good to go.
+  * Note-2: It's alright to discover lacking privileges later on - you can remedy it at the point.
```

Comment:
I don't know if i understand that sentence after reading up to this point so maybe remove it?
```suggestion
  * Note-2: It's alright to discover lacking privileges later on - you can remedy it at the point.
```